mod_form.php

```diff
@@ -41,6 +41,22 @@
  */
 class mod_moodleoverflow_mod_form extends moodleform_mod {
 
+    /**
+     * constructor
+     * @see moodleform_mod::moodleform_mod
+     */
```

Comment:
I think the parent already does this could you double-check if this is necessary? :)

Reply:
yes, it is unnecessary :)